database/insertData/_common/03_template_C_userRegistration.js

```diff
@@ -58,6 +58,10 @@ exports.queries = [
                       title: "Username"
                       elementTypePluginCode: "shortText"
                       category: QUESTION
+                      parameters: { 
+                        label: "Select a username" 
+                        description: "Usernames should have no spaces and lenght of 3-50 using letters, numbers or - . _"
```

Comment:
I think this could be simplified to something like: "description: You can use letters, numbers and - . _  (min 3 chars)"

Just in the interests of simplicity -- no need to mention maxLength, hardly anyone will ever reach it.

I just checked a couple of real sites for how they specify this restriction. Here's Google's:
![Screen Shot 2021-06-10 at 12 18 35 PM](https://user-images.githubusercontent.com/5456533/121445790-427bc200-c9e6-11eb-98c3-bbd36f339cdb.png)

Also -- should we accept "@" character, as some people like to use their email address as a username.

Reply:
Great. Changing to this: " You can use letters, numbers and - . _  or @" : )